tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -38,15 +38,23 @@ sub run {
 
     #	bootloader options and set probe foreign OS, timeout
     assert_and_click 'yast2-bootloader_bootloader-options';
+
+    wait_still_screen 10;
+    send_key 'alt-t';    # Import untrusted GnuPG key -- workaround
```

Comment:
not needed if we change the priority of the child job or if that doesn't work creating a new test module to zypper ref with the param to auto accept gpg keys.

Reply:
added a new module: zypper_ref

Reply:
you need to pass zypper ref the parameter to accept the gpg keys, for which I would not touch this module used everywhere.

Reply:
I've added `--gpg-auto-import-keys` in zypper_ref.pm

Reply:
it is not reflected in your code changes...

Reply:
Do you mean I should use `zypper_call('--gpg-auto-import-keys ref')` in the existing module?

Reply:
better create a new module `zypper_ref_auto_import_keys.pm` so we could use it regardless the order of the test modules in the test suite when this happen sporadically. But you didn't try to adjust yet the priority, I think that was the option number 1 we decided instead of this.

Reply:
I changed the priority to 20.

Reply:
great, thanks! please always paste the link for referenced PRs/MRs
https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/354/